installscripts/cookbooks/jenkins/files/jobs/job_trigger_platform_services_build.sh

```diff
@@ -0,0 +1,72 @@
+JENKINS_URL=http://$1/ # localhost or jenkins elb url
+JOB_NAME="Trigger_Platform_Services_Build"
```

Comment:
This is specific to gitlabs, so name it appropriately. Perhaps we should also try to separate code specific to each SCM.

Reply:
In cooperated the comment